lib/csv_report_generator.rb

```diff
@@ -0,0 +1,42 @@
+class CsvReportGenerator
+  CSV_PATH = "/data/uploads/publisher/reports"
```

Comment:
It would be better for this to be a directory under the `Rails.root` that can then be symlinked to the NFS mount at deploy time.  See asset-manager for prior art on this.